src/main/java/frc/robot/subsystems/drive/GyroIOPigeon2.java

```diff
@@ -39,12 +41,15 @@ public GyroIOPigeon2() {
     yawPositionQueue = OdometryThread.getInstance().registerSignal(yaw::getValueAsDouble);
     accelX.setUpdateFrequency(50.0);
     accelY.setUpdateFrequency(50.0);
+    RobotPitch.setUpdateFrequency(50.0);
+    RobotRoll.setUpdateFrequency(50.0);
```

Comment:
You can simplify this to `BaseStatusSignal.setUpdateFrequencyForAll(50.0, statusSignal1, ...)`